kin-sdk/Environment.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using kin_base;
+
+namespace kin_sdk
+{
+    public class Environment
+    {
+
+        public readonly string networkUrl;
+        public readonly string networkPassphrase;
+        
+        /// <summary>
+        /// Build an Environment object.
+        /// </summary>
+        /// <param name="networkUrl"> The URL of the blockchain's horizon </param>
+        /// <param name="networkPassphrase"> The passphrase/network id to use</param>
+        public Environment(string networkUrl, string networkPassphrase)
+        {
+            Uri uriResult;
+            if (!(Uri.TryCreate(networkUrl, UriKind.Absolute, out uriResult) &&
+               (uriResult.Scheme == Uri.UriSchemeHttp || uriResult.Scheme == Uri.UriSchemeHttps)))
+            {
+                throw new ArgumentException($"networkUrl {networkUrl ?? ""} is not a valid URL");
+            }
+
+            if (String.IsNullOrEmpty(networkPassphrase))
+            {
+                throw new ArgumentException($"networkPassphrase cannot be null or empty");
+            }
+
+            this.networkUrl = networkUrl;
+            this.networkPassphrase = networkPassphrase;
+        }
+
+        public static readonly Environment Production = new Environment("https://horizon.kinfederation.com","Kin Mainnet ; December 2018");
+
+        public static readonly Environment Test = new Environment("https://horizon-testnet.kininfrastructure.com",
+        "Kin Testnet ; December 2018");
+
+        public bool IsMainNet()
+        {
+            return Production.networkUrl == this.networkUrl.Replace("http://", "https://") && Production.networkPassphrase == this.networkPassphrase;
+        }

```

Comment:
This helper function can be moved to an extension class for Environment.  I also think helper methods like this should be placed in a library that couples all unrelated libraries.

```
    public static class EnvironmentExtensions 
    {
        public static bool IsMainNet(this Environment env)
        {
            return Production.networkUrl == env.networkUrl.Replace("http://", "https://") && Production.networkPassphrase == env.networkPassphrase;
        }

    }
```